src/color-swatch/color-swatch.js

```diff
@@ -143,6 +165,28 @@ const Self = class ColorSwatch extends NudeElement {
 			},
 			reflect: false,
 		},
+		coords: {
+			type: {
+				is: Array,
+				values: {
+					is: Object,
+					defaultKey: (value, i) => {
+						let [space, channel] = value.split(".");
+						if (!channel) {
+							channel = space;
+							space = this.color?.space.id;
+						}
+
+						return Color.Space.get(space)?.coords[channel]?.name ?? channel;
+					},
+				},
+			},
+			default: [],
+			reflect: {
+				from: true,
+			},
+			dependencies: ["color"],
```

Comment:
So you found a bug in Nude Element: that `defaultKey()` (and `defaultValue()`) isn't taken into account when calculating dependencies. Don't just work around bugs, file them so we can fix them!

Reply:
You think about me too well! 😂 I didn't work around the bug; I knew we infer dependencies from `default`, `get`, and `convert`. And for me, it's just like that. So, I just worked with the current implementation. I'm happy that thanks to my “discovery,” we'll make `NudeElement` even better, though. 🙂

Reply:
ANY tool you use can have bugs. That includes popular libraries, frameworks, even the browser itself. Always try to report them. And here we can even fix them at the source!